recipes/agent_windows.rb

```diff
@@ -1,3 +1,32 @@
 # Installs and configures the New Relic Infrastructure agent on Windows
+#
+# Chef uses sha256 checksums, to generate this, I downloaded the file
+# and on OS X did 'shasum -a 256 newrelic-infra.msi' YMMV on other
+# platforms
+#
+# TODO: pin to specific version
+# TODO: abstract checksum
+# TODO: make install location configurable
 
-raise 'Windows chef support for New Relic Infrastructure agent not yet available'
+windows_package 'newrelic-infra' do
+  action :install
+  source 'https://download.newrelic.com/infrastructure_agent/windows/newrelic-infra.msi'
+  checksum '3c9f98325dc484ee8735f01b913803eaef54f06641348b3dd9f3c0b3cd803ace'
```

Comment:
That is correct, in our case we wrote a wrapper around this cookbook and did something like the following in the wrapper cookbooks `attributes/default.rb`

```
# pin to a specific version
node.override['newrelic-infra']['agent_version'] = "1.0.690-1"
```
and after computing the hash of the file:
```
node.override['newrelic-infra']['checksum'] = "3c9f98325dc484ee8735f01b913803eaef54f06641348b3dd9f3c0b3cd803ace"
```

Would you like me to update this PR in any way prior to merging?